src/tools/ScenarioMeasurement/Startup/Startup.csproj

```diff
@@ -18,4 +25,17 @@
     <ProjectReference Include="..\..\Reporting\Reporting\Reporting.csproj" />
   </ItemGroup>
 
+  <Target Name="DownloadPerfCollect" BeforeTargets="CoreCompile" Condition=" '$(OS)' != 'Windows_NT' ">
```

Comment:
>'$(OS)' != 'Windows_NT' [](start = 77, length = 23)

Is this a standard msbuild property for .NET Core or is this set by Helix somewhere?

Reply:
This is a standard msbuild property so it works on client runs as well.